linode/manager
# upcoming: [M3-7619] - Unassign Linode from PG Modal

## Description 📝
This PR bring the unassign modal to the Placement Groups > Linodes flow

## Changes  🔄
- Add and connect unassign modal to PG flow
- Add test for unassign modal

## Preview 📷
![Screenshot 2024-02-12 at 10 58 37](https://github.com/linode/manager/assets/130582365/b79b7f60-f7f1-49e3-ba5b-5c9647b281fb)


## How to test 🧪

### Prerequisites
- Turn on both the "Placement Groups" feature flag and MSW

### Verification steps 
- Navigate to `http://localhost:3000/placement-groups/1/linodes`
- Click on `Unassign` on any of the Linodes
- Confirm the modal (**NOTE**:  because of MSW, the linode title in the modal will be "DC-Specific Pricing Linode" **every time**)
- Click on unassign and confirm payload

## As an Author I have considered 🤔

*Check all that apply*
